apps/studio/components/layouts/ProjectLayout/ResizingState.tsx

```diff
@@ -24,13 +24,15 @@ export const ResizingState = () => {
         <div className="space-y-6 py-6">
           <div className="flex px-8 space-x-8">
             <div className="mt-1">
-              <Loader2 className="animate-spin" size={18} />
+              <Loader2 className="animate-spin text-foreground-light" size={18} />
             </div>
-            <div className="space-y-1">
-              <p>Resizing project...</p>
+            <div className="flex flex-col gap-1">
+              <p>Resizing Project Compute size</p>
```

Comment:
just to check on the consistency - are we capitalizing "project"? seems like we're doing it in some places, but not all?

Reply:
what about "Compute" -> are we capitalizing? we should make it consistent everywhere in the dashboard

Reply:
IMO we should capitalize. A quick AI explanation...

<img width="955" alt="Screenshot 2024-10-31 at 17 58 27" src="https://github.com/user-attachments/assets/885574dd-8f96-4be6-bc77-9fbf751772b9">